src/utils/displayCultuurkuurLabels.ts

```diff
@@ -0,0 +1,16 @@
+import { PermissionTypes } from '@/constants/PermissionTypes';
+
+const displayCultuurkuurLabels = (roles: string[], labels: string[]) => {
+  const isGodUser = roles?.includes(PermissionTypes.GEBRUIKERS_BEHEREN);
+
+  const cultuurkuurLabels = labels.filter((label) =>
+    label.startsWith('cultuurkuur_'),
+  );
+  const otherLabels = labels.filter(
+    (label) => !label.startsWith('cultuurkuur_'),
+  );
+
+  return isGodUser ? [...cultuurkuurLabels, ...otherLabels] : [...otherLabels];
+};
```

Comment:
You don't need to spread `otherLabels` since it's already an array.
```suggestion
  return isGodUser ? [...cultuurkuurLabels, ...otherLabels] : otherLabels;
};
```